package.json

```diff
@@ -88,6 +89,7 @@
     "loader-utils": "2.0.4",
     "node-forge": "1.3.0",
     "nth-check": "^2.0.1",
+    "path-to-regexp": "0.1.10",
```

Comment:
maybe just include this one for now, and follow up on the next one?

Reply:
Sounds good. Thad and I confirmed that functionality is restored with just this change